testing/test_conftest.py

```diff
@@ -402,13 +405,13 @@ def pytest_addoption(parser):
 
 
 class TestConftestVisibility:
-    def _setup_tree(self, testdir):  # for issue616
+    def _setup_tree(self, pytester: Pytester) -> None:  # for issue616
```

Comment:
> testing/test_conftest.py:468: error: No return value expected [return-value]

You have given return type as ```None```. But this function is actually returning a ```Dict```. Make sure to ```Dict``` using ```from typing import Dict```.
```suggestion
    def _setup_tree(self, pytester: Pytester) -> Dict[str, Path]:  # for issue616
```